src/pokemon_summary_screen.c

```diff
@@ -97,24 +97,24 @@ static EWRAM_DATA struct PssData
     u8 currPageIndex;
     u8 minPageIndex;
     u8 maxPageIndex;
-    bool8 unk40C3;
+    bool8 lockMonFlag; // This is used to prevent the player from changing pokemon in the move deleter select, etc, but it is not needed because the input is handled differently there
     u16 newMove;
     u8 firstMoveIndex;
     u8 secondMoveIndex;
     bool8 unk40C8;
-    u8 unk40C9;
+    u8 bgDisplayOrder; // Determines the order page backgrounds are loaded while scrolling between them
     u8 filler40CA;
     u8 windowIds[8];
     u8 spriteIds[28];
     bool8 unk40EF;
-    s16 unk40F0;
+    s16 switchCounter; // Used for various switch statement cases that decompress/load graphics or pokemon data
     u8 unk_filler4[6];
 } *pssData = NULL;
-EWRAM_DATA u8 gUnknown_0203CF20 = 0;
-static EWRAM_DATA u8 gUnknown_0203CF21 = 0;
-static EWRAM_DATA u8 gFiller_0203CF22[2] = {0};
+EWRAM_DATA u8 gLastViewedMonIndex = 0;
+static EWRAM_DATA u8 sUnknown_0203CF21 = 0;
+static EWRAM_DATA u8 sFiller_0203CF22[2] = {0};
```

Comment:
This looks like `ALIGNED(4)`, i.e. `ALIGNED(4) static EWRAM_DATA u8 sUnknownTaskId = 0;` Try to clean up 'filler' declarations if you can.